.evg.yml

```diff
@@ -102,9 +103,124 @@ tasks:
     commands:
       - func: "generate sbom"
       - func: "upload sbom"
+      - func: "push SBOM Lite to Silk"
       - func: "scan sbom"
 
+  - name: ssdlc-artifacts
+    run_on: ubuntu2204-small
+    depends_on:
+      - name: sbom
+        variant: code-quality-and-correctness
+    exec_timeout_secs: 300 # 5m
+    commands:
+      - func: "pull augmented SBOM from Silk"
+      - func: "publish augmented SBOM"
```

Comment:
The final publishing of the augmented SBOM to the release bucket should be part of the Release task so that we don't publish it if the release fails.